docs/bugs.json

```
@@ -1,4 +1,14 @@
 [
+    {
+        "uid": "SOL-2023-3",
+        "name": "VerbatimInvalidDeduplication",
+        "summary": "``verbatim`` calls with different data can wrongfully be de-duplicated if used in very similar context.",
+        "description": "The block deduplicator is an optimizer step which identifies equivalent assembly blocks and merges them into a single one. However when blocks contained ``verbatim``, their comparison was implemented incorrectly, leading to the collapse of assembly blocks which are identical except on the contents of the ``verbatim`` items. Since ``verbatim`` is only available in Yul, compilation of Solidity sources is not affected.",
```

Just a few small tweaks:

```suggestion
        "description": "The block deduplicator is a step of the opcode-based optimizer which identifies equivalent assembly blocks and merges them into a single one. However, when blocks contained ``verbatim``, their comparison was performed incorrectly, leading to the collapse of assembly blocks which are identical except for the contents of the ``verbatim`` items. Since ``verbatim`` is only available in Yul, compilation of Solidity sources is not affected.",
```